x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -215,6 +215,8 @@ export const getNetworkTopNFlowColumns = (
   },
 ];
 
+// Xavier fix this stuff under DO NOT FORGET
```

Comment:
Is this a `TODO` for a future PR or is it leftover?

Reply:
It will go away with the host stuff!!!

Reply:
I get mixed up with these two

Reply:
I fix it in the host  stuff

Reply:
Ha, understandable -- sounds good, just wanted to check :)